nix/kexec-installer/test.nix

```diff
@@ -137,6 +137,8 @@ makeTest' {
     node1.succeed('/root/kexec/kexec --version >&2')
     node1.succeed('/root/kexec/run >&2')
 
+    time.sleep(6)
```

Comment:
What is the purpose of this one?

Reply:
Since the kexec-run also sleeps for 6 seconds this just reduces annoying log noise a bit.